components/AsanaCard/index.js

```diff
@@ -1,33 +1,95 @@
 import styled from "styled-components";
+import Image from "next/image";
+import Check from "../../public/Images/noun-check-mark-1550146.svg";
+import Close from "../../public/Images/noun-close-1386186.svg";
 
 export default function AsanaCard({asana}) {
   return (
-    <article>
-      <Image
-        width={2000}
-        height={5000}
+    <Article>
+      <StyledImage
+        width="200"
+        height="600"
         src={asana.image}
-        alt={`yoga pose: ${asana.name}`}
+        alt={`yoga pose: ${asana.sanskrit}`}
         priority
       />
       <StyledAsanaCard>{AsanaCard}</StyledAsanaCard>
-      <h2>{asana.name}</h2>
-    </article>
+      <StyledSanskrit>{asana.sanskrit}</StyledSanskrit>
+      <StyledSanskrit>{asana.sanskrit2}</StyledSanskrit>
+      <ButtonContainer>
+        <Button>
+          <Image
+            style={{position: "relative", top: "8px"}}
+            src={Close}
+            width="40"
+            height="40"
+            alt={"close"}
+          />
+        </Button>
+        <ButtonContent></ButtonContent>
+        <Button>
+          <Image
+            style={{position: "relative", top: "8px"}}
+            src={Check}
+            width="40"
+            height="40"
+            alt={"checkmark"}
+          />
+        </Button>
+      </ButtonContainer>
+    </Article>
   );
 }
 
 const StyledAsanaCard = styled.article`
   margin: 0;
-  padding: 3rem 3rem 3rem 3rem;
-  min-height: calc(100vh - 8rem);
-  margin-top: 0;
+  margin-top: 10px;
 `;
 
-const Image = styled.img`
-  padding: 1rem 1rem 1rem 1rem;
-  border-radius: 120px;
-  box-shadow: 0 2px 5px 2px rgba(0, 0, 0, 0.45);
-  width: 600px;
-  height: 1000px;
-  margin: 45%;
+const StyledImage = styled(Image)`
+  padding: 18rem 1rem 0 1rem;
+  width: 300px;
```

Comment:
You StyledImage also have a property called width. It would be better to set the width on one way and not on two different ways.